src/main/webapp/WEB-INF/web.xml

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE web-app PUBLIC
+ "-//Sun Microsystems, Inc.//DTD Web Application 2.3//EN"
+ "http://java.sun.com/dtd/web-app_2_3.dtd" >
```

Comment:
web-app_2_3?  what is DTD and how does that compare to other Java web-app deployment descriptors?  Is this an old webapp version? be careful with this value, because it can change how you have to load your application context when trying to spin up compontents for later unit testing.